index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./style.css" />
+		<link
+			href="https://fonts.googleapis.com/css?family=Poppins"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>My portfolio</title>
+	</head>
+	<body>
+		<section id="header">
+			<div class="logo-placeholder">
+				<p class="logo">SHAH</p>
+			</div>
+			<div class="menu">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
+		</section>
+		<section id="headline_section">
+			<div class="headline">
+				<div>
+					<h2>I'm Shah Zaman. Glad to see you!</h2>
```

Comment:
Kindly, I suggest you use an `h1` tag instead of `h2` to follow HTML best practices. 😄